src/taco-remote/lib/buildManager.ts

```diff
@@ -369,10 +369,20 @@ class BuildManager {
         if (fs.existsSync(changeListFile)) {
             buildInfo.changeList = JSON.parse(fs.readFileSync(changeListFile, { encoding: "utf-8" }));
             if (buildInfo.changeList) {
+                buildInfo.changeList.deletedFiles = buildInfo.changeList.deletedFiles.map(function (deletedFile: string): string {
+                    // Convert all \ and / characters in the path string into platform-appropriate path separators
+                    return deletedFile.split("\\").join(path.sep).split("/").join(path.sep);
+                });
                 buildInfo.changeList.deletedFiles.forEach(function (deletedFile: string): void {
-                    if (!deletedFile.match(/^plugins[\/]/)) {
+                    
+                    if (deletedFile.split(path.sep)[0] === "plugins") {
```

Comment:
shouldn't the comparison be !== if we want to skip plugins scenario? how do we catch scenario where request is foo\..\plugins\?